src/markdown-pages/404/404-ar.md

```diff
@@ -0,0 +1,16 @@
+---
+lang: ar
+templateKey: 404-page
+slug: "404"
+pageTitle: "ar 404"
+error: ar Page not found
+causeOfError: ar We’re sorry, we can’t find the page you're looking for. It might have been removed, changed its name, or is otherwise unavailable.
```

Comment:
- As I mention in another comment, it might make sense to just use the markdown body for this text.
- When you do use the front matter, quotation marks are not needed around strings. 
- I'm not sure we need the ways to fix error section. I know it's in the template 404 page, but our URLs can follow multiple potential patterns
- Also for the visit our home page section, I'm not sure if we should include the 'helpful tools and resources part'

Reply:
* Addressed in the other comment, can ignore that here, unless a change is made.
* In this case, quotation marks are needed because there is a type error. For example, I cannot have `home` and `404` for slugs, as they are different types (I was getting an error when I did it like this). After making `404` a string, the error was resolved.
* It might be an accessibility thing that we should take into account. We can keep it like this for now, and discuss it during code review?
* We should also discuss this during code review.

Reply:
Oh okay the type error makes sense. Maybe the slug should be `not-found` and the page title can be "Page Not Found" or something along those lines?

Reply:
Gatsby tends to use `404` as the slug as we saw in my code in `gatsby-node.js` which was taken from Gatsby documentation.
```javascript
// Check if the page is a localized 404 (i.e. /en/404/)
        if (edge.node.fields.slug.match(/^\/[a-z]{2}\/404\/$/)) {
```

Reply:
Okay sounds good